Orange/canvas/config.py

```diff
@@ -44,6 +45,11 @@ def init():
     QCoreApplication.setApplicationVersion(version)
     QSettings.setDefaultFormat(QSettings.IniFormat)
 
+    # Hack to make sure the correct plugin search path is added. (Github issue #1143)
+    for path in site.getsitepackages():
+        if path.endswith("site-packages"):
+            QCoreApplication.addLibraryPath(os.path.join(path, "PyQt4", "plugins"))
```

Comment:
It's not ok to simply set the default library path and effectively override any user's set preferred path, if one exists. It would probably be better to _extend_ the library path with the one in site-packages. I think you can do it like:
```py
QCoreApplication.setLibraryPaths(
    QCoreApplication.libraryPaths() + [os.path.join(path, "PyQt4", "plugins")])
```
Does this work?